build/test/httpserver_test.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+# Test the Easegress' basic functionality which is generating
+# an HTTPServer and Pipeline for testing HTTP Requests.
+# author:   benjaminwu 
+# date:     2021/0716
+
+# path related define.
+SCRIPTPATH=`pwd -P`
+pushd $SCRIPTPATH"/../../example" > /dev/null
+EXAMPDIR=`pwd -P`
+WRITER01DIR=$EXAMPDIR"/writer-001"
```

Comment:
`SCRIPTPATH=$(dirname $(realpath ${BASH_SOURCE[0]}))`

`pwd -P` isn't always the script's path.

If we locate script path, the example path should be got by relative path to scriptpath

Reply:
Updated as comment.